chainerx_cc/chainerx/routines/normalization.cc

```diff
@@ -178,9 +178,6 @@ std::tuple<Array, std::unique_ptr<BatchNormGradState>> GenericBatchNormOp::Call(
         const Axes& axis,
         bool return_state,
         const nonstd::optional<Array>& out) {
-    CHAINERX_ASSERT(internal::GetArrayBody(x)->nodes().empty());
-    CHAINERX_ASSERT(internal::GetArrayBody(gamma)->nodes().empty());
-    CHAINERX_ASSERT(internal::GetArrayBody(beta)->nodes().empty());
```

Comment:
Oops, I overlooked these asserts. Device implementations have expected arrays without nodes. If this assumption is unchanged, we should use `AsGradStopped` as before.

Reply:
Reverted the change.